apps/filecoin-site/src/app/_components/LogoSection/LogoItem.tsx

```diff
@@ -3,22 +3,23 @@ import type { ComponentType, SVGProps } from 'react'
 export type LogoItemProps = {
   logo: ComponentType<SVGProps<SVGSVGElement>>
   alt: string
-  height?: number
+  size?: number
   href?: string
 }
 
 export function LogoItem({
   logo: SvgComponent,
   alt,
-  height = 40,
+  size = 40,
   href,
 }: LogoItemProps) {
   const logoElement = (
     <SvgComponent
-      height={height}
+      height={size}
+      width="auto"
       aria-label={href ? undefined : alt}
       aria-hidden={href ? 'true' : undefined}
-      className="h-auto max-w-full"
+      className="max-w-full text-zinc-400"
```

Comment:
I think we want to add a class here like `logo-section-logo-item` and then in `globals.ccs` adapt states based on whether they appear `dark-section` or `light-section`.

For example:

```css
  .split-section-description-text {
    @apply text-lg leading-7 text-balance;

    .dark-section & {
      @apply text-zinc-50;
    }

    .light-section &,
    .gray-section & {
      @apply text-zinc-950;
    }
```

Reply:
Oh - Didn't Filipa say during standup today that she wanted the logos to be `dark-zinc-50` on hover on dark? Like in the footer?